docs/introduction/path_to_chrysalis.md

```diff
@@ -8,81 +8,77 @@ Progress in Coordicide research has resulted in identifying many concepts that c
 
 The benefit of this approach is that many of the protocol properties will remain the same, or almost the same, for Coordicide. That, together with a better set of developer tools, will make the transition to Coordicide much simpler.
 
```

Comment:
**For line 5** I'd add a comma after 2016
```suggestion 
The IOTA mainnet has been operational since 2016, and the overall engineering strategy evolved significantly based on industry demand and feedback.
```